packages/manager/modules/pci/src/projects/new/payment/components/register/translations/Messages_en_GB.json

```diff
@@ -1,9 +1,3 @@
 {
-  "pci_project_new_payment_register_title": "Register a payment method",
-  "pci_project_new_payment_register_explain": "You need to register a payment method to validate your identity.  You will not be debited and nothing will be withdrawn until you consume resources. ",
```

Comment:
Removed in https://github.com/ovh/manager/commit/73d802b70d3602c624841e5f1044131d29904deb#diff-d01ef5d4749e219ffaadc97e748dbab4